client/utils/contentProfiles.ts

```diff
@@ -81,10 +82,57 @@ export function getUnusedProfileFields(
     profile: IEditorProfile,
     includeGroupCheck: boolean = true
 ): Array<IProfileFieldEntry> {
+    // Temporary FIXME: Will be removed once we add support for CVs as fields in coverage profiles
+    if (profile.name === 'coverage') {
+        const fieldsFromProfile = getProfileFields(profile);
+
+        return orderBy(
+            fieldsFromProfile
+                .filter((field) => (includeGroupCheck && field.field.group == null) || !field.field.enabled),
+            'name'
+        );
+    }
+
+    const cvs = planningApi.vocabularies.getCustomVocabularies();
+    const cvIds = cvs.map((x) => x._id);
+    const cvFieldEntries = cvs
+        .map((x, i) => ({
+            field: {
+                enabled: false,
+                group: undefined,
+                index: i,
+            },
+            name: x._id,
+            schema: {
+                type: 'custom_vocabulary',
+                required: false,
+            }
+        }) as IProfileFieldEntry);
```

Comment:
It looks like you're casting when you don't need to. Either use `const result: IProfileFieldEntry = ...` or `satisfies` keyword.